src/main/java/com/fasterxml/jackson/databind/SerializationFeature.java

```diff
@@ -237,6 +237,18 @@
      */
     WRITE_DURATIONS_AS_TIMESTAMPS(true),
     
+    /**
+     * Feature that determines whether time values are serialized with a
+     * fraction part or not.
+     *<p>
+     * Note: if enabled the feature {@link
+     * #WRITE_DATE_TIMESTAMPS_AS_NANOSECONDS} is ignored in {@link
```

Comment:
I think some mention of the new flag should be part of the javadoc for WRITE_TIMESTAMPS_AS_NANOSECONDS..